server/www/templates/posts/views/category-select.html

```diff
@@ -0,0 +1,10 @@
+<fieldset ng-show="categories.length">
+    <legend translate="nav.categories">Categories</legend>
+
+    <div class="form-field checkbox" ng-repeat="(index, category) in categories">
+        <label>
+            <input checklist-value="category.id" checklist-model="model" type="checkbox" name="selectedCategories" > {{ ::category.name }}
```

Comment:
For the checklist-model, when you want to clear the filters checklist model UI may not respect model=[]
https://github.com/vitalets/checklist-model#usage
I used their suggested work around of splice but there may be a better way to do it.